server/coordinator.go

```diff
@@ -211,8 +211,12 @@ func (c *coordinator) addOperator(op Operator) bool {
 	}
 	if op.GetResourceKind() == priorityKind {
 		o, ok := c.operators[regionID]
-		if ok && o.GetResourceKind() == priorityKind {
-			return false
+		if ok {
+			if o.GetResourceKind() == priorityKind {
+				return false
+			}
+			c.limiter.removeOperator(o)
+			log.Infof("coordinator: add more prioriy operator: %+v remove operator: %+v", op, o)
```

Comment:
I think put this block inside `if _, ok := c.operators[regionID]; ok` will make it more clear.

```
if old, ok = c.operator[regionId]; ok {
        if op.GetResourceKind() == priorityKind && old.GetResourceKind() != priorityKind {
                c.limiter. removeOperator(old)
                log.Infof("coordinator: add operator with higher priority: %+v remove operator: %+v", op, o)
        } else {
                return false
        }
}

c.limiter.addOperator(op)
...
```